### DISCUSSION TITLE
v0.24.0 (2024-06-08)

### DISCUSSION THREAD

**Discussion by mtkennerly** [mtkennerly/ludusavi#347] [Announcements]
* Added:
  * On the "other" screen, you can choose not to back up games with cloud support on certain stores. If a game is customized or already has local backups, then Ludusavi will continue backing it up regardless.
  * For Heroic roots, Ludusavi now supports Amazon and sideloaded games.
  * For Lutris roots, Ludusavi now scans `pga.db` in addition to `games/*.yml` in case the YAML files do not contain all of the necessary information.
  * CLI: There is a new `api` command that can be used for bulk queries. Right now, it only supports looking up titles (analogous to the `find` command).
  * CLI: There is a new `schema` command to display some of Ludusavi's schemas.
  * CLI: The `find` command now accepts a `--lutris-id` option.
  * CLI: The `backups` command output now includes each game's backup directory.
* Changed:
  * Title normalization now ignores apostrophes and quotation marks (e.g., `ludusavi find --normalized "Mirrors Edge"` will find `Mirror's Edge`).
  * Some additional fields in the config file have been made optional.
  * Updated translations. (Thanks to contributors on the [Crowdin project](https://crowdin.com/project/ludusavi))
* Fixed:
  * For Heroic and Lutris roots, if you had multiple copies of the same game, Ludusavi would only use the metadata from one of them.
  * GUI: The game-level duplication badge did not always fade out when the conflicts were resolved.

<hr /><em>This discussion was created from the release <a href='https://github.com/mtkennerly/ludusavi/releases/tag/v0.24.0'>v0.24.0 (2024-06-08)</a>.</em>

**Comment by Sae3sy** [comment]
does ludusavi-manifest also contain the information if a game supports cloud save on a certain store?

**Reply by mtkennerly** [reply]
Yes, I recently added a `cloud` field for that information :) Here's an example:

```yaml
Elden Ring:
  cloud:
    steam: true
```

And here's the relevant part of the schema: https://github.com/mtkennerly/ludusavi-manifest/blob/dde621a1f1b1b751d9bf4124bf1940d23bda9a1c/data/schema.yaml#L110-L122

**Reply by Sae3sy** [reply]
that's cool! I'm currently coding an backup application and I have been thinking lately about exact the same function - to don't backup games that are/can be saved arleady in a cloud! Although I wanted to do it only for online games which anyways ignore local save data if the data isn't exact the same on the server so there would be no point in backuping them. If there is a info if a game is a online game on PCGamingWiki could you add it to ludusavi-manifest?

**Reply by mtkennerly** [reply]
I checked a couple of articles ([Genshin Impact](https://www.pcgamingwiki.com/wiki/Genshin_Impact) and [World of Warcraft](https://www.pcgamingwiki.com/wiki/World_of_Warcraft)), but I couldn't find any metadata/category to indicate that the save data is managed centrally. If you'd like, you could try asking on the PCGamingWiki Discord if they're open to adding a category or something for this - if so, I'd be happy to update the manifest to use it.

**Reply by Sae3sy** [reply]
I also could not find any list/category that would match it... But theoretically it must be games that you can find in a list of games which are [using always online drm](https://www.pcgamingwiki.com/wiki/List_of_games_using_Always_Online_DRM) and a list of [online multiplayer games](https://www.pcgamingwiki.com/wiki/List_of_online_multiplayer_games). And to make sure that these are only online games the games should not have "singleplayer" listed under the "modes" section. I don't belive that games with these "specifications" don't load the save data from a server. Can you do something with it? Or doesn't it make sense? Couldn't you based on this information define in the manifest if a game is online-only?
